phantomgames/Home/HomeScreenViewController.swift

```diff
@@ -25,19 +25,19 @@ class HomeScreenViewController: UIViewController{
         viewModel.fetchHomeResults()
     }
     private func setupTableView() {
-        tableView.register(CustomHomeTableViewCell.HometableViewNib(), forCellReuseIdentifier: Constants.TableViewIdentifiers.customHomeCellIdentifier)
+        tableView.register(CustomHomeTableViewCell.hometableViewNib(), forCellReuseIdentifier: Constants.TableViewIdentifiers.customHomeCellIdentifier)
         tableView.delegate = self
         tableView.dataSource = self
     }
-    
     private func setupCollectionView() {
         homeCollectionView.dataSource = self
         homeCollectionView.register(UINib(nibName: "CustomCollectionViewCell", bundle: nil), forCellWithReuseIdentifier: "CustomCollectionViewCell")
     }
 }
+
 // MARK:  Collection View
 
-extension HomeScreenViewController: UICollectionViewDataSource{
+extension HomeScreenViewController: UICollectionViewDataSource, UICollectionViewDelegate,UICollectionViewDelegateFlowLayout {
```

Comment:
HomeScreenViewController: UICollectionViewDataSource, UICollectionViewDelegate, UICollectionViewDelegateFlowLayout 
Swiftlint should be throwing a warning about no space between the comma. Please let me know if the warning is not showing or if it being ignored?